### ISSUE TITLE
Build failure: lunarvim

### ISSUE THREAD

**Issue by MisileLab** [NixOS/nixpkgs#349782]
### Steps To Reproduce

1. build *lunarvim* `nix build github:nixos/nixpkgs/9fd64fc3499912546ce76ad1b4dae6d017f98b07#lunarvim`

### Build log

```
error:
       … while calling the 'derivationStrict' builtin
         at <nix/derivation-internal.nix>:34:12:
           33|
           34|   strict = derivationStrict drvAttrs;
             |            ^
           35|

       … while evaluating derivation 'lunarvim-1.4.0'
         whose name attribute is located at /nix/store/02kyk6ji56wvs3ic6pngqk3x86jrkanz-source/pkgs/stdenv/generic/make-derivation.nix:336:7

       … while evaluating attribute 'installPhase' of derivation 'lunarvim-1.4.0'
         at /nix/store/02kyk6ji56wvs3ic6pngqk3x86jrkanz-source/pkgs/by-name/lu/lunarvim/package.nix:101:3:
          100|
          101|   installPhase = ''
             |   ^
          102|     runHook preInstall

       (stack trace truncated; use '--show-trace' to show the full, detailed trace)

       error: 'utillinux' has been renamed to/replaced by 'util-linux'
```

### Notify maintainers

@ProminentRetail @lebensterben 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.4, NixOS, 24.11 (Vicuna), 24.11.20241018.c4a6d1b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/8p6g5kry9shcjl00pc9gr763bm8hqkcc-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by MisileLab** [comment]
[log with --show-trace](https://gist.github.com/MisileLab/989a81612b631cc32c70f54592374488)

**Comment by MisileLab** [comment]
close when https://github.com/NixOS/nixpkgs/pull/349783

**Comment by lebensterben** [comment]
Thanks for reporting, although this is not a bug caused by `lunarvim` package per se.

This build failure is *scheduled* to happen on Oct 17 2024:
https://github.com/NixOS/nixpkgs/blob/73d8d52149461e2b70a1ad2b90fd7cbb0e16c267/pkgs/top-level/aliases.nix#L1676

closed by #349783
